src/iomei.cpp

```diff
@@ -168,7 +168,10 @@ bool MEIOutput::Export()
     try {
         pugi::xml_document meiDoc;
 
-        if (m_page < 0) {
+        // Saving the entire document
+        // * With score-based MEI, all mdivs are saved
+        // * With page-based MEI, only visible mdivs are saved
+        if (!this->IsSavingSinglePage()) {
```

Comment:
It would make sense to use this function in some other places, i.e. in `MEIOutput::WriteObject`